docs/api/browser-view.md

```diff
@@ -41,7 +41,7 @@ Objects created with `new BrowserView` have the following properties:
 
 #### `view.webContents` _Experimental_
 
-A [`WebContents`](web-contents.md) object owned by this view.
+A `WebContents | null` property that returns the `WebContents` owned by this view or `null` if the contents are [destroyed](web-contents.md#event-destroyed). 
```

Comment:
Looks like just this one trailing space and this is good to go!

Reply:
Thanks for catching that! I've fixed the issue so this should be good to go